_sass/layout/_l-banner.scss

```diff
@@ -45,8 +45,16 @@ $backgrounds: 'promoHome', 'promoInteriores-About', 'promoInteriores-CompareCoun
 
     > .wrap {
       display: flex;
-      justify-content: center;
-      flex-direction: column;
+      justify-content: space-between;
+      flex-direction: row;
+    }
+
+    > .wrap > .js--country-silhouette {
+
+      > svg {
+        fill: $beige;
+        opacity: .3;
+      }
```

Comment:
I know this os not too much code, but I think you should make a new module to fit the scss structure